app/code/Magento/Sales/Model/AdminOrder/Create.php

```diff
@@ -984,6 +984,7 @@ public function applySidebarData($data)
                 );
                 if ($item->getId()) {
                     $this->addProduct($item->getProduct(), $item->getBuyRequest()->toArray());
+                    $data['remove'][$itemId] = 'wishlist';
```

Comment:
What if `$data['remove']` does not exist?

Reply:
In this case we put value under key that will be processed on line 996. I have rewritten that code to be more readable.